modules/gradle-plugin/src/main/kotlin/org/jetbrains/compose/storytale/plugin/Utils.kt

```diff
@@ -101,11 +102,11 @@ val KotlinCompilation<*>.resolveDependencyResourcesTaskName: String
   get() = "${name.lowercase()}${target.name.capitalized()}ResolveDependencyResources"
 
 fun setupResourceResolvingForTarget(storytaleBuildDir: File, compilation: KotlinCompilation<*>) {
-  compilation.project.tasks.register<ResolveResourcesFromDependenciesTask>(compilation.resolveDependencyResourcesTaskName) {
+  compilation.project.tasks.register(compilation.resolveDependencyResourcesTaskName, ResolveResourcesFromDependenciesTask::class.java, Action {
```

Comment:
for some reason after the files into a new package, the previous call didn't want to compile - the errors was "unresolved symbol".

The new call is doing the same.